src/wp-admin/export.php

```diff
@@ -25,34 +26,36 @@ function export_add_js() {
 ?>
 <script type="text/javascript">
 	jQuery(document).ready(function($){
- 		var form = $('#export-filters'),
- 			filters = form.find('.export-filters');
- 		filters.hide();
- 		form.find('input:radio').change(function() {
+		 var form = $('#export-filters'),
+			 filters = form.find('.export-filters');
+		 filters.hide();
+		 form.find('input:radio').change(function() {
 			filters.slideUp('fast');
 			switch ( $(this).val() ) {
 				case 'attachment': $('#attachment-filters').slideDown(); break;
 				case 'posts': $('#post-filters').slideDown(); break;
 				case 'pages': $('#page-filters').slideDown(); break;
 			}
- 		});
+		 });
```

Comment:
Same as above, the extra whitespaces don't get removed, just moved after the tab.